### PR TITLE
Relegate document.loadingMode to a potential future

### PR DESCRIPTION
**Pull request by domenic**
For now, the prerendering state API and other existing APIs should suffice. Related to #2.

@bokand, your review would be appreciated!

### CODE REVIEW

File: browsing-context.md [hunk 1]
```diff
@@ -240,23 +183,37 @@ if (geoPermission.state === "denied") {
...
 _Note: the above code only makes sense if we decide that permissions are denied in prerendering browsing contexts, instead of having them hang until activation. [That plan](#restrictions-on-the-basis-of-being-non-user-visible) is still tentative._
 
-Finally, there's the case of the [page visibility API](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API). Our current plan is to treat (non-portal) prerendering browsing contexts as hidden, until activation, in which case code that only wants to run upon the user viewing the page could be done as follows:
+Finally, for cases related to rendering behavior, we propose a [dedicated prerendering state API](./prerendering-state.md):
 
 ```js
-if (document.visibilityState === "visible") {
-    videoElement.play();
+function afterPrerendering() {
+  // grab user data from cookies/IndexedDB
```
Comment (bokand) [inline browsing-context.md:190]: This and requesting notifications access seems better covered by the storage and permissions APIs above. More relevant to prerendering could be "start a video/animation", "fetch large resources", "connect to chat server", etc.

**Review by bokand — APPROVED** [review]
One suggestion but LGTM!

File: browsing-context.md (hunk 1 above)
Comment (domenic) [reply browsing-context.md:190]: Good call; leftover from some copy and pasting!

**Comment by jeremyroman** [comment]
lgtm2

**Pull request merged**